gui/assets/css/overrides.css

```diff
@@ -9,7 +9,7 @@
 
 body {
     padding-bottom: 70px;
-    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: "Helvetica Neue", Calibri, Helvetica, Arial, sans-serif;
```

Comment:
Windows aliases Helvetica to Arial. If Helvetica is before Calibri in the
font stack Windows will render using Arial.

On Mon, Nov 23, 2015, 11:37 AM Antony Male <notifications@github.com> wrote:

> In gui/assets/css/overrides.css
> <https://github.com/syncthing/syncthing/pull/2508#discussion_r45625722>:
>
> > @@ -9,7 +9,7 @@
> >
> >  body {
> >      padding-bottom: 70px;
> > -    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
> > +    font-family: "Helvetica Neue", Calibri, Helvetica, Arial, sans-serif;
>
> Should we have Helvetica before Calibri?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syncthing/syncthing/pull/2508/files#r45625722>.
>